nav2_waypoint_follower/src/waypoint_follower.cpp

```diff
@@ -304,4 +354,52 @@ WaypointFollower::goalResponseCallback(
   }
 }
 
+
+std::vector<geometry_msgs::msg::PoseStamped>
+WaypointFollower::convertGPSWaypointstoPosesinMap(
+  const
+  std::vector<sensor_msgs::msg::NavSatFix> & gps_waypoints,
+  const rclcpp_lifecycle::LifecycleNode::SharedPtr & parent_node,
+  nav2_util::ServiceClient<robot_localization::srv::FromLL> & fromll_client)
+{
+  RCLCPP_INFO(parent_node->get_logger(), "Converting GPS waypoints to Map Frame..");
+
+  std::vector<geometry_msgs::msg::PoseStamped> poses_in_map_frame_vector;
+  int index_of_gps_waypoints = 0;
```

Comment:
The variable `index_of_gps_waypoints` doesn't appear to be used anywhere. Its incremented on success / failure but not used anywhere. Can this be removed?